packages/trader/src/Stores/Modules/Trading/Actions/contract-type.ts

```diff
@@ -2,33 +2,61 @@ import { ContractType } from 'Stores/Modules/Trading/Helpers/contract-type';
 import { TTradeStore } from 'Types';
 
 type TOnChangeContractTypeList = (store: TTradeStore) => {
-    contract_type: string;
+    contract_type: string | number;
 };
 
 type TContractValues = Pick<
     TTradeStore,
     | 'form_components'
     | 'basis_list'
-    | 'basis'
-    | 'trade_types'
-    | 'start_date'
-    | 'start_dates_list'
-    | 'contract_start_type'
-    | 'barrier_count'
-    | 'barrier_1'
-    | 'barrier_2'
-    | 'duration_unit'
     | 'duration_units_list'
-    | 'duration_min_max'
     | 'expiry_type'
     | 'accumulator_range_list'
-    | 'multiplier_range_list'
     | 'cancellation_range_list'
->;
+> & {
+    basis: string | number;
+    trade_types: string[];
+    start_date: number;
+    start_dates_list: (
```

Comment:
@kate-deriv please add missing properties to TOverrideTradeStore which exists in this branch and can be accessed here via TTradeStore. no need to write this big type here, everything can be picked from TTradeStore :)

Reply:
I think I can remove it at all